config/vufind/config.ini

```diff
@@ -354,6 +354,10 @@ title_level_holds_mode = "disabled"
 ; Whether support for multiple library cards is enabled. Default is false.
 ;library_cards = true
 
+; Whether support for connecting multiple library cards using Shibboleth is
+; enabled. You have also to enable Shibboleth authentication. Default is false.
```

Comment:
I added a TODO point to the task. I'd suggest creating the wiki page and adding a link to it here. Something like "See https://vufind.org/wiki/configuration:shibboleth_library_cards for more information".

Reply:
Good idea. Please let me know if you need me to create that page (or increase anyone's wiki rights so that they can do it). (Page creation is a bit restricted right now due to spammers abusing it).